src/scripts/components/colour/Panels.js

```diff
@@ -161,12 +199,13 @@ export default class Panels extends Component {
             <Tab name='Most visited'>
               <ul className='panels__panel'>
                 { state.topSites.map((site, i) => (
-                  <li key={i}>
-                    <a className={`item-${i}`} title={site.title} href={site.url}
-                      style={{ backgroundImage: `url('${site.img}')` }}>
-                      {site.title}
-                    </a>
-                  </li>
+                    <li key={i}>
+                      <a className={`item-${i}`} title={site.title} href={site.url}
+                        style={{ backgroundImage: `url('${site.img}')` }}>
+                        {site.title}
+                        <button className={`item-${i}--remove`} data-url={site.url} onClick={this.blacklist}>Remove</button>
```

Comment:
Also why use the data attribute instead of just having it directly as the argument for the `blacklist` function?